css-two-column-layout/styles.css

```diff
@@ -0,0 +1,87 @@
+.container {
+  max-width: 1100px;
+  margin: auto;
+}
+
+.row {
+  display: flex;
+}
+
+.column-fourth {
+  flex-basis: calc(100% / 2);
+}
+
+.row2 {
+  display: flex;
+}
+
+.column-third {
+  flex-basis: calc(100% / 3);
+}
+
+.column-two-thirds {
+  flex-basis: calc(200% / 3);
+}
+
+.rowformat {
+  justify-content: space-around;
+  flex-wrap: wrap;
+}
+
+.colthirdformat {
+  padding: 5px 20px;
+}
+
+.col2thirdformat {
+  padding-top: 20px;
+}
+
+.colfourthformat {
+  padding: 30px;
+}
+
+.row2format {
+  flex-wrap: wrap;
+  padding: 10px;
+}
```

Comment:
These class names are lazy. Give it a name that describes either what it does (if it is a utility class) or what it is being applied to (if it is a semantic class)

Reply:
fixed #1 and #2. not sure if I did #3 correctly.